devolving_music/models/song_submission.py

```diff
@@ -31,7 +50,61 @@ def voteable(self) -> bool:
     def get_voteable_submissions(Event):
         voteable_submissions = [
                 sub
-                for sub in SongSubmission.objects.filter(event__exact=Event).order_by('?')
+                for sub in SongSubmission.objects.filter(event__exact=Event).order_by('id')
                 if sub.voteable()
             ]
         return voteable_submissions
+
+    @staticmethod
+    def check_compare(comparison_submission:"SongComparison",\
```

Comment:
Insufficiently descriptive method name.

Spaces after commas and colons.

You don't need the backslash.